tests/csv_46.t

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+. bash_tap_fd.sh
+
+PASSWD=$(get_password)
+CMD="$(get_script_python3) --format csv"
+TEST="$(get_test_data)/test_profile_firefox_46/"
+
+${bashtap_org_pwd}/diff_csv.py <(echo ${PASSWD} | ${CMD} ${TEST} | grep doesntexist || kill $$) <(get_user_data "doesntexist_csv")
```

Comment:
A better test would be to test different `--delimiter` and `--quotechar` from default settings.
This ensures those two arguments are working as expected.